bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/handler/ApiBridgeHandler.java

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.linky.internal.handler;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Hashtable;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.linky.internal.LinkyAuthServlet;
+import org.openhab.binding.linky.internal.LinkyBindingConstants;
+import org.openhab.binding.linky.internal.LinkyException;
+import org.openhab.binding.linky.internal.dto.Contracts;
+import org.openhab.binding.linky.internal.dto.CustomerIdResponse;
+import org.openhab.binding.linky.internal.dto.CustomerReponse;
+import org.openhab.binding.linky.internal.dto.IdentityInfo;
+import org.openhab.core.auth.client.oauth2.AccessTokenResponse;
+import org.openhab.core.auth.client.oauth2.OAuthClientService;
+import org.openhab.core.auth.client.oauth2.OAuthException;
+import org.openhab.core.auth.client.oauth2.OAuthFactory;
+import org.openhab.core.auth.client.oauth2.OAuthResponseException;
+import org.openhab.core.config.core.Configuration;
+import org.openhab.core.io.net.http.HttpClientFactory;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingRegistry;
+import org.openhab.core.thing.ThingStatus;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * {@link ApiBridgeHandler} is the base handler to access enedis data.
+ *
+ * @author Laurent Arnal - Initial contribution
+ */
+@NonNullByDefault
+public abstract class ApiBridgeHandler extends LinkyBridgeHandler {
+    private final Logger logger = LoggerFactory.getLogger(ApiBridgeHandler.class);
+
+    private final OAuthClientService oAuthService;
+
+    private static @Nullable HttpServlet servlet;
+
+    private static final String TEMPLATE_PATH = "templates/";
```

Comment:
This is just my habit, but I like to have `static final` sorted above `final`s

Reply:
ok. notes that this declaration was moved to LinkyAuthServlet.